OpenMEEG/libs/OpenMEEG/src/sensors.cpp

```diff
@@ -225,7 +227,7 @@ namespace OpenMEEG {
     void Sensors::findInjectionTriangles() {
         om_error(m_geo!=NULL);
         m_weights = Vector(m_positions.nlin());
-        m_weights.set(0.);
+        m_weights.set(1.);
```

Comment:
no, each weight was overwritten ... but I've set the default case for the 3-columns EIT case